x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/add_domain_form_errors.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { AddDomainLogic } from './add_domain_logic';
+
+export const AddDomainFormErrors: React.FC = () => {
+  const { errors } = useValues(AddDomainLogic);
+
+  if (errors.length > 0) {
+    return (
+      <EuiCallOut
```

Comment:
Why do we use an inline EuiCallOut here rather than flash messages?

I'm guessing it's because we want to show the error in the modal, not in the main page body?

Reply:
Maybe this would make a worthwhile generic component at some point in the future...

```
<FormErrors title="some title" errors={errors} />
```